language/tools/move-cli/src/lib.rs

```diff
@@ -132,6 +126,7 @@ impl OnDiskStateView {
         let mut path = self.get_addr_path(module_id.address());
         path.push(MODULES_DIR);
         path.push(module_id.name().to_string());
+        path.set_extension(MOVE_COMPILED_EXTENSION);
```

Comment:
Lol @me storing this with no extension before. We should probably make up an extension for resource files and event log files too...